src/main/resources/db/changelog/001-create-p-order.yaml

```
@@ -0,0 +1,59 @@
+              - column: { name: type, type: varchar(20), constraints: { nullable: false } }
+              - column: { name: status, type: varchar(30), constraints: { nullable: false } }
+              - column: { name: shipping_fee, type: bigint, constraints: { nullable: false } }
+              - column: { name: total_amount, type: bigint, constraints: { nullable: false } }
+
+              # 사유 (nullable)
+              - column: { name: cancel_reason, type: varchar(200) }
+              - column: { name: return_reason, type: varchar(200) }
+              - column: { name: reject_reason, type: varchar(200) }
+
+              # 비즈니스 시간 (nullable, CONFIRMED 전이 시 채움)
+              - column: { name: confirmed_at, type: timestamp }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)